pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -33,6 +34,8 @@ const (
 	cacheTTL = 10 * time.Second
 )
 
+var SharedWithMeFolderPermission = accesscontrol.Permission{Action: "folders:read", Scope: "folders:uid:" + folder.SharedFolderUID}
```

Comment:
nit: better use [ActionFoldersRead](https://github.com/grafana/grafana/blob/a98a39c418de86385c2f15157481c1692b103f72/pkg/services/dashboards/accesscontrol.go#L16) and [ScopeFoldersPrefix](https://github.com/grafana/grafana/blob/a98a39c418de86385c2f15157481c1692b103f72/pkg/services/dashboards/accesscontrol.go#L13) constants